lib/cmd/fhc/target.js

```diff
@@ -137,7 +137,7 @@ function undefinedTargCleanup(cb, tar) {
 }
 
 function pingTarget(targ, callback) {
-  request(targ + "/box/srv/1.1/act/sys/auth/logout", function(err, response, body){
```

Comment:
@jasonmadigan @david-martin that shouldn't be a problem with current master. I removed the CSRF checks from the LogouBean: https://github.com/fheng/millicore/commit/e16b483b8985961b94dd5ca25d25301006a30957